### PR TITLE
fix: Increase strategies limit

### PR DESCRIPTION
**Pull request by ChaituVR**
- This will increase the strategies limit from `8` to `10`
- Also adds a check for strategies length on `get_vp`

Refer https://discord.com/channels/955773041898573854/1187284357799481484/1188191395261972640

### CODE REVIEW

File: src/rpc.ts [hunk 1]
```diff
@@ -39,8 +39,14 @@ router.post('/', async (req, res) => {
 
   if (method === 'get_vp') {
     if (params.space && disabled.includes(params.space))
-      return rpcError(res, 429, 'too many requests', null);
-
+      return rpcError(res, 429, 'too many requests', id);
+    if (
+      !params.strategies ||
+      params.strategies.length === 0 ||
+      params.strategies.length > 10
```
Comment (wa0x6e) [inline src/rpc.ts:46]: We could extract `10` to a constant, and avoid having to edit test files in case of update

File: src/utils.test.ts [hunk 2]
```diff
@@ -258,13 +259,13 @@ describe('formatStrategies function', () => {
     expect(formattedStrategies).toHaveLength(2);
   });
 
-  it('should limit strategies to 8', () => {
-    const strategies = new Array(10).fill({ name: 'strategy', param: 'a' });
+  it(`should limit strategies to ${MAX_STRATEGIES}`, () => {
+    const strategies = new Array(12).fill({ name: 'strategy', param: 'a' });
```
Comment (wa0x6e) [inline src/utils.test.ts:263]: Could use something like `MAX_STRATEGIES + 1`

**Review by wa0x6e — CHANGES_REQUESTED** [review]

**Review by wa0x6e — APPROVED** [review]
utAck

File: src/utils.test.ts (hunk 2 above)
Comment (ChaituVR) [reply src/utils.test.ts]: Nice catch 👍

**Pull request merged**